doc/source/aw_harburg_de.md

```diff
@@ -0,0 +1,126 @@
+# AW Harburg
+
+Support for schedules provided by [AW Landkreis Harburg](https://www.landkreis-harburg.de) located in Lower Saxony, Germany.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: aw_harburg
+      args:
+          district_level_1: "Hanstedt"
+          district_level_2: "Evendorf"
+```
+
+### Configuration Variables
+
+**district_level_1**<br>
+*(string) (required)*
+
+**district_level_2**<br>
+*(string) (required)*
+
+**district_level_3**<br>
+*(string) (optional - depending on district_level_2)*
+
+## Example
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: aw_harburg
+      args:
+          district_level_1: "Buchholz"
+          district_level_2: "Buchholz mit Steinbeck (ohne Reindorf)"
+          district_level_3: "Seppenser Mühlenweg Haus-Nr. 1 / 2"
+      customize:
+        - type: Biotonne
+          alias: Biomüll
+          show: true
+        - type: Grünabfall
+          alias: Grünabfall
+          show: true
+        - type: Gelber Sack
+          alias: Gelber Sack
+          show: true
+        - type: Hausmüll 14-täglich
+          alias: Hausmüll 2wö
+          show: true
+        - type: Hausmüll 4-wöchentlich
+          alias: Hausmüll 4wö
+          show: true
+        - type: Altpapier
+          alias: Papier
+          show: true
+```
+
+Use `sources.customize` to filter or rename the waste types:
```

Comment:
Is there a difference in the 2nd example compared to the first? If not, please remove it.